README.md

```diff
@@ -72,40 +100,30 @@ Evaluators are individuals able to evaluate the technology delivered as a result
-   6. Sign off on the [terms and conditions](https://github.com/w3f/Open-Grants-Program/blob/master/src/T&Cs.md) presented by [claassistantio](https://github.com/claassistantio) as Contributor License Agreement (CLA). You might need to reload the pull request to see the comment by [claassistantio](https://github.com/claassistantio).
-2. **Application Review:**
+   5. Once you're done, create a pull request. The pull request should only contain _one new file_—the Markdown file you created from the template.
+   6. Sign off on the [terms and conditions](https://github.com/w3f/Open-Grants-Program/blob/master/src/T&Cs.md) presented by the [CLA assistant](https://github.com/claassistantio) bot as a Contributor License Agreement. You might need to reload the pull request to see its comment.
+2. **Application Review**
    1. The [committee](#w3f-grants-committee) can (and usually does) issue comments and request changes on the pull request.
-   2. The application is accepted when all requested changes are addressed, the terms and conditions have been agreed upon and one third of the committee approves your pull request.
-3. **Milestone Delivery:**
-   1. [Fork](https://github.com/w3f/Grant-Milestone-Delivery/fork) the [Grant Milestone Delivery](https://github.com/w3f/Grant-Milestone-Delivery) repo with the **same GitHub account, which you used to submit the application!**
-   2. In the newly created fork, create a **copy** of the [milestone-delivery-template.md](https://github.com/w3f/Grant-Milestone-Delivery/blob/master/deliveries/milestone-delivery-template.md).
-   3. Label the file as **`project_name_milestone_number.md`**.
-   4. Fill out the [invoice form](https://forms.gle/QvsbeSA3ziuC9k9b9).
-   5. Fill out the template with the details of your milestone including a **link to the initial pull request**.
-   6. Once you have completed the application, click on "Create new pull request".
-4. **Milestone Review:**
-   1. [Evaluators](#w3f-grants-evaluators) can (and usually do) issue comments and request changes on the pull request.
-   2. Grants Evaluators will merge your pull request to accept the deliverable.
-5. **Milestone Payment:**
-   1. The [Operations Team](#w3f-operations-team) receives a notification once the delivery was accepted. As soon as feedback is provided by the evaluators, this feedback first needs to be resolved.
-   2. Operations Team makes the payment to the Bitcoin or Ethereum address specified in the initial application. The exchange rate for BTC depends on the time of payment.
-
-_The grant application can be amended at any time. However, this necessitates a reevaluation by the Committee._
-
-Additionally, we try to enforce the [code of conduct](https://github.com/w3f/Open-Grants-Program/blob/master/CODE_OF_CONDUCT.md) and based on this may [block users](https://github.blog/2016-04-04-organizations-can-now-block-abusive-users/).
```

Comment:
Is this information actually in the delivery readme?

Reply:
just checked. It is, but the links are dead because they are relative.

Reply:
Thanks for noticing, will fix it.